### ISSUE TITLE
Incorrect invocation of `strcmp()` for URL matching

### ISSUE THREAD

**Issue by dslc** [Molorius/esp32-websocket#16]
Thanks for the library. I noticed that the invocation of `strcmp()` in the [`ws_server_send_text_clients_from_callback()`](https://github.com/Molorius/esp32-websocket/blob/master/websocket_server.c#L339) function is incorrect.

The relevant test should be
```c
if(ws_is_connected(clients[i]) && !strcmp(clients[i].url,url)) {
```
since `strcmp()` returns zero on match.

**Comment by Molorius** [comment]
You're right, and it should also check that clients[i].url and url is not NULL first or there will be a segfault. Thanks for pointing this out, I will try to fix it soon.

**Comment by Molorius** [comment]
Fixed in commit 5e417abf4e4a40cc5b7868849fdb5bd03c36dece, thank you very much.
